index.html

```diff
@@ -34,6 +34,7 @@
       <p><a href="https://docs.wpilib.org/">FIRST Robotics Competition Control System documentation</a></p>
       <p><a href="allwpilib/docs/release/java/">WPILib Java API Docs</a></p>
       <p><a href="allwpilib/docs/release/cpp/">WPILib C++ API Docs</a></p>
+      <p><a href="https://robotpy.readthedocs.io/projects/robotpy/en/latest/">WPILib Python API Docs</a></p>    
```

Comment:
Since we link to the release docs for Java and C++, should do the same for Python

```suggestion
      <p><a href="https://robotpy.readthedocs.io/projects/robotpy/en/stable/">WPILib Python API Docs</a></p>    
```